### PR TITLE
Journal week 3 both a and b

### PR DESCRIPTION
**Pull request by jmcreasman**

### CODE REVIEW

**Comment by cmauban** [comment]
Good job on completing all of the assignments! :+1: For your tutorial... your way is a great starting point. Definitely understanding what every CSS property did first was useful and made things a lot easier while creating our CSS files.

**Comment by ShayHall** [comment]
Good job.  CSS is a sneaky bugger, isn't it?  One small note "inline' is one word not two.  :+1: If you could shoot :dart: a review my way, that would be great. https://github.com/ShayHall/shayhall.github.io/pull/6

**Comment by jeffworley** [comment]
Jon, Excellent writing!  I like that you connected the two journal entries and they came across informative and genuine.  You sir get a :+1:

File: tutorial-week-3.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Title
+
+Since this is suppose to be a tutorial based essay I thought I’d share how I go about completing the weekend homework when I feel like I can barely grasp the subject. I hope that this is helpful for anyone who might feel like they are falling behind on any given week’s topic. As I mentioned briefly in my other essay for this week I will be using the example of last weekend’s topic/homework (CSS).
```
Comment (kellymurray) [inline tutorial-week-3.md:3]: this is a big subject! You might want to pick something a little simpler - maybe how to do one thing in CSS?

**Comment by Ricecoder** [comment]
:+1: Great job! I thought your journal was funny and your tutorial informative.

**Comment by ericksonk** [comment]
I like that your tutorial is on a simple topic but something we all struggle with. I want to try this method out for sure. Also, the tldr; is much appreciated :+1:

**Comment by jeffworley** [comment]
Excellent refactoring of your entries.  the tl:dr at the end of the tutorial was a brilliant use of that concept.  :+1:

**Comment by jeremyrist** [comment]
:+1: I'm reminded of the great slogan on the front of [_Tommy Boy_:](http://www.imdb.com/title/tt0114694/?ref_=ttqt_qt_tt)
_If at first you don't succeed, lower your standards._

Not really in the spirit at all of what you were saying but that's a funny movie. I hear ya on the goal-oriented approach to learning! That can definitely help, as well as the extra time reading up on something once you've already started to learn it.

File: journal-week-3.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+##My Love Hate/Affair with CSS
+
+After two weeks of having JavaScript smash my poor brain into mush I heard seven little words the made my day.
+
+“This week we’ll be focusing on CSS”.
+
+“THANK GOD!” - I thought to myself - “I know CSS, I’ve used it before. Heck I’ve taken the codecademy course, how hard can this be right?” Let’s pause for a moment of silence for my poor, poor misguided past self. Because as I would soon find out there is a BIG difference between using CSS to change a color here and there and using CSS extensively to create and position a whole page.
```
Comment (kellymurray) [inline journal-week-3.md:7]: :laughing:

**Comment by jeremyrist** [comment]
:+1: Good work Jon!

**Comment by ericquayson** [comment]
Good journal. I can relate to your frustration about CSS. :+1:

**Comment by boshnivolo** [comment]
Very relatable journal. I feel like it definitely correlates with what we worked on today :+1:

File: journal-week-3.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+##My Love Hate/Affair with CSS
+
+After two weeks of having JavaScript smash my poor brain into mush I heard seven little words that made my day.
+
+“This week we’ll be focusing on CSS”.
+
+“THANK GOD!” - I thought to myself - “I know CSS, I’ve used it before. Heck I’ve taken the codecademy course, how hard can this be right?” Let’s pause for a moment of silence for my poor, poor misguided past self. Because as I would soon find out there is a BIG difference between using CSS to change a color here and there and using CSS extensively to create and position a whole page.
```
Comment (kellymurray) [inline journal-week-3.md:7]: :laughing:

**Pull request merged**